dashboard/agent.py

```diff
@@ -186,6 +189,12 @@ async def _check_parent():
         required=True,
         type=int,
         help="The port to expose metrics through Prometheus.")
+    parser.add_argument(
+        "--metrics-agent-port",
```

Comment:
I think it's better to call it `--dashboard-agent-port`, because many agent grpc services will share this port. For example, `reporter agent`, `job agent`, ...